README.md

```diff
@@ -7,44 +7,95 @@
 
 Solutions Template is a boilerplate template for building repeatable
 solutions with the best practices in architecture on Google Cloud, including GKE
-clusters, Test Automation, CI/CD, as well as development process.
+clusters, CloudRun, Test Automation, CI/CD, as well as development process.
 
 This template provides built-in and ready-to-ship sample features including:
-* Kubernetes-based microservices
+* Container-based microservices, can be deployed to a Kubernetes cluster or CloudRun.
 * Simplified deployment using Skaffold and Kustomize
-* Terraform Google Cloud foundation setup
+* Google Cloud foundation setup using Terraform
 * CI/CD deployment (with Github Actions)
 * CloudRun templates
-* [In Progress] CI/CD deployment with CloudBuild
 
 ## Roadmap
 
-Please see Feature Requests in the Github issue list at https://github.com/GoogleCloudPlatform/solutions-template/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22
+Please see [Feature Requests in the Github issue list](https://github.com/GoogleCloudPlatform/solutions-template/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22).
 
-## Getting Started
+## Prerequisites
 
-### Prerequisites
+### Understanding Google Cloud
 
-Install Cookiecutter ([Github](https://github.com/cookiecutter/cookiecutter)):
+We recommend the following resources to get familiar with Google Cloud and microservices.
+
+- What is Microservice Architecture: https://cloud.google.com/learn/what-is-microservices-architecture
+- Kubernetes:
+  - Learn about the basics of Kubernetes: https://kubernetes.io/docs/concepts/overview/
+  - Google Kubernetes Engine (GKE) overview: https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview
+  - Skaffold, a command line tool that facilitates continuous development for container based & Kubernetes applications: https://skaffold.dev/docs/
+- CloudRun:
+  - Serverless container deployment and execution with CloudRun: https://cloud.google.com/run/docs/overview/what-is-cloud-run
```

Comment:
The hyper links can be embedded:
 
- What is [Microservice Architecture](https://cloud.google.com/learn/what-is-microservices-architecture)
- Kubernetes:
  - Learn about the basics of [Kubernetes](https://kubernetes.io/docs/concepts/overview/)
  - [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview) overview
  - [Skaffold](https://skaffold.dev/docs/), a command line tool that facilitates continuous development for container based & Kubernetes applications
- Cloud Run:
  - Serverless container deployment and execution with [Cloud Run](https://cloud.google.com/run/docs/overview/what-is-cloud-run)

Reply:
Resolved.